src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -841,6 +843,219 @@ def test_WINNF_FT_S_SIQ_11(self):
       self.assertFalse('availableChannel' in response[response_num])
       self.assertEqual(response[response_num]['response']['responseCode'], 300)
 
+  def generate_SIQ_12_default_config(self, filename):
+    """Generates the WinnForum configuration for SIQ.12."""
+    # Load GWBL information
+    gwpz_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'gwpz_record_0.json')))
+    # Load device info
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_e.json')))
+
+    # Load PAL records
+    pal_record1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+    pal_record2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'pal_record_1.json')))
+
+    # Load PPA records
+    ppa_record1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'ppa_record_0.json')))
+    ppa_record2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'ppa_record_1.json')))
+    
+    pal_low_frequency1 = 3570000000
+    pal_high_frequency1 = 3580000000
+    pal_user_id1 = "pal_user_1"
+    pal_low_frequency2 = 3600000000
+    pal_high_frequency2 = 3610000000
+    pal_user_id2 = "pal_user_2"
+
+    # Make PAL and PPA record consistent
+    ppa_record1, pal_record1 = makePpaAndPalRecordsConsistent(
+        ppa_record1, [pal_record1], pal_low_frequency1, pal_high_frequency1,
+        pal_user_id1)
+    ppa_record2, pal_record2 = makePpaAndPalRecordsConsistent(
+        ppa_record2, [pal_record2], pal_low_frequency2, pal_high_frequency2,
+        pal_user_id2) 
+
+    # Move device_b near to the first PPA zone
```

Comment:
Why is it located near a PPA zone? Does it mean you want to restrict its GAA frequencies?

Reply:
latitude,longitude   of device_b parameters are updated to be within PPA zone